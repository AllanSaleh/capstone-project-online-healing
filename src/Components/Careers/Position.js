import React from 'react';

export default function Position(props) {
  return (
    <div className="flex justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4">
      <div>
        <div className="text-blue-dark text-subtitle">{props.title1}</div>
        <div className="text-gray-700 text-paragraph">{props.subtitle1}</div>
      </div>

      <div className="flex flex-col">
        <div className="text-subtitle text-blue-dark">{props.title2}</div>
        <div className="text-gray-700 text-paragraph">{props.subtitle2}</div>
      </div>
    </div>
  );
}
