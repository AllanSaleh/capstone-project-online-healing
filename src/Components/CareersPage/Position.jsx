import React from 'react';

export default function Position({ title1, subtitle1, title2, subtitle2 }) {
  return (
    <div className="flex flex-col justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4">
      <div>
        <div className="text-blue-dark text-lg lg:text-subtitle">{title1}</div>
        <div className="text-gray-700 text-base lg:text-paragraph">{subtitle1}</div>
      </div>

      <div className="flex flex-col">
        <div className="text-lg lg:text-subtitle text-blue-dark">{title2}</div>
        <div className="text-gray-700 text-base lg:text-paragraph">{subtitle2}</div>
      </div>
    </div>
  );
}
