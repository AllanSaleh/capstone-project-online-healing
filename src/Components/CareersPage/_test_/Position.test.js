import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Position from '../Position';

it(' Position renders correctly', () => {
  const tree = renderer.create(<Position />).toJSON();
  expect(tree).toMatchSnapshot();
});
