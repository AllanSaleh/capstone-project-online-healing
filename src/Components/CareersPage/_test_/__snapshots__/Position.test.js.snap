// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col justify-between w-11/12 border-b-2 border-gray-700 mb-6 py-4"
>
  <div>
    <div
      className="text-blue-dark text-lg lg:text-subtitle"
    />
    <div
      className="text-gray-700 text-base lg:text-paragraph"
    />
  </div>
  <div
    className="flex flex-col"
  >
    <div
      className="text-lg lg:text-subtitle text-blue-dark"
    />
    <div
      className="text-gray-700 text-base lg:text-paragraph"
    />
  </div>
</div>
`;
